src/main.js

```diff
@@ -212,8 +212,13 @@ $(document).ready(() => {
       $toggler.mouseenter(() => toggleSidebar(true));
 
       // Immediately closes if click outside the sidebar.
-      $document.on('click', () => {
-        if (!isMouseInSidebar && !isSidebarPinned() && isSidebarVisible()) {
+      $document.on('click', (event) => {
+        /**
+         * Ensure the event click being triggered outside the sidebar
+         */
+        const notInSidebar = !$sidebar.find(event.target).length;
```

Comment:
So we're not using the var 'isMouseInSidebar' anymore. Options:

1. Should remove it.
2. Should fix the code so that isMouseInSidebar has the right value, so that doesn't have to change this piece